docs/spec_draft.md

```diff
@@ -127,3 +127,183 @@ specification.
 %z = stablehlo.add %x, %y : tensor<2x2xf32>
 // %z: [[6, 8], [10, 12]]
 ```
+
+## stablehlo.and
+
+`stablehlo.and(lhs, rhs) -> result`
+
+### Semantics
+
+An element-wise operation performing bitwise AND of two tensors `lhs` and `rhs`
```

Comment:
"Performs element-wise..." to be consistent with `add`, `maximum` and `minimum`?

Reply:
done.